shell/core/plugin-routes.ts

```diff
@@ -86,9 +84,16 @@ export class PluginRoutes {
       }
     });
 
-    const matcher = createRouterMatcher([...orderedPluginRoutes, ...allRoutes], {});
+    if ( orderedPluginRoutes.length === 0) {
+      return;
+    }
+
+    // Remove all existing routes. Once we upgrade our router version we'll have access to clearRoutes() https://router.vuejs.org/api/interfaces/Router.html#clearRoutes
```

Comment:
`clearRoutes()` is written in the documentation but is not included in the latest library version.

Reply:
I'd like to leave the comment in case it becomes available and someone stumbles across this.

However, it may get entirely removed by https://github.com/rancher/ui-plugin-examples/compare/main...feature/11706-update-vue3